bin/gce-delete-instance

```diff
@@ -1,9 +1,48 @@
 #!/bin/bash
 
-uage="Usage: `basename $0` <instance_name> [-q | --quite]"
+delete-instance() {
```

Comment:
The original ``gce-delete-instance`` is wrapped in the function ``delete-instance``.